azure-servicefabric/azure/servicefabric/credentials.py

```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
```

Comment:
Update your Autorest, if you still have this file, means your Autorest is old.

Reply:
Should be fixed now, ran with Autorest 1.2.2, but that also ended up modifying the legal headers. Not sure if there's a script you can run to fix them.